synthea_omop_etl/models/omop/schema.yml

```diff
@@ -1,5 +1,12 @@
 version: 2
 
+sources:
+  - name: synthea
+    database: dbt_synthea
+    schema: synthea
```

Comment:
per dbt docs, it is not necessary to re-specify the `schema` param here since it's the same as the `name`: https://docs.getdbt.com/docs/build/sources#declaring-a-source